AccentColorizer/main.cpp

```diff
@@ -40,15 +42,21 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
 			//  d) Device was turned on after sleep, and colors and bitmaps probably were reset
 			g_dwAccent = NULL;
 		}
+		if (message == WM_DWMCOLORIZATIONCOLORCHANGED || (message == WM_WTSSESSION_CHANGE && wParam == WTS_SESSION_UNLOCK)) {
+			accentColorChanges++;
+		}
+		else {
```

Comment:
Don't we need to re-apply colorization if theme has been changed (`WM_THEMECHANGED`)? I'm not sure, just wondering

Reply:
This is used when you change the accent color in order to find your previous saturation. You can check ```AccentColorHelper.cpp``` for why I'm using it.